modules/qna/src/translations/fr.json

```diff
@@ -40,15 +40,17 @@
     "cantBeSaved": "Ne peut être enregistré",
     "chatbotWillRandomlyChoose": "L'agent conversationnel répondra au hasard parmi ces alternatives",
     "confirmDeleteQuestion": "Êtes-vous certain de vouloir supprimer cette question? Les questions et réponses alternatives seront aussi supprimé.",
+    "copyIdToClipboard": "Copier le ID au presse-papiers",
     "deleteQuestion": "Supprimer la question",
     "disabledTooltip": "Ce question-réponses ne sera pas affiché aux utilisateurs.",
     "disableQuestion": "Désactiver la question",
-    "disableRedirect": "Désactiver la redirection",
+    "disableRedirection": "Désactiver la redirection",
     "duplicateAnswer": "Vous avez déjà écrit cette réponse",
     "duplicateQuestion": "Vous avez déjà écrit cette question",
     "emptyState": "Appuyez sur le + dans la bar d'outils pour ajouter votre première question.",
     "enableQuestion": "Activer la question",
-    "enableRedirect": "Activer la redirection",
+    "enableRedirection": "Activer la redirection",
+    "idCopiedToClipboard": "ID copié au presse-papiers",
```

Comment:
Isn't it "dans le presse-papiers" ?

Reply:
![Screen Shot 2020-10-13 at 7 29 44 AM](https://user-images.githubusercontent.com/3555826/95874598-e614b580-0d25-11eb-8631-a8d491aafd3c.png)
yeah as Yann said, just to stay consistent with the rest of the app

Reply:
Fixed in e403dd2c36676b290424b89152786d36068ddd62